pkg/controller/appdefinition/testdata/volumes/cluster-default-with-bind-combos/input.yaml

```diff
@@ -34,12 +34,13 @@ status:
       bar:
         class: test-custom-class-request
         size: 6Gi
-  defaults:
+  resolvedOfferings:
     volumes:
       foo:
-        class: test-custom-class-default
-        size: 2Gi
-        accessModes: [ "readWriteMany" ]
+        class: test-custom-class
+        size: 3Gi
+        accessModes: [ "readWriteOnce" ]
```

Comment:
So that it matches what is defined in `spec.volumes` higher up in the file. By the time we reach this unit test, offerings should already be resolved, so it doesn't make sense for our input file to have them resolved incorrectly.